core/src/cluster_info.rs

```diff
@@ -1679,9 +1682,18 @@ impl ClusterInfo {
         let now = timestamp();
         let self_id = me.id();
 
+        //skip messages that are likely to be pushed
+        let min_filter_time = me.start_time + 10 * CRDS_GOSSIP_PULL_CRDS_TIMEOUT_MS;
+        let push_timer = if min_filter_time < now {
```

Comment:
Aren't push messages are generated roughly every `GOSSIP_SLEEP_MILLIS` through `run_gossip() -> gossip_request()`, not through `push_self()`,  which is only for `ContactInfo`?